olive/data/container/data_container.py

```diff
@@ -30,6 +31,8 @@ def load_dataset(self, data_root_path: Optional[str] = None):
         """Run load dataset."""
         params_config = self.config.load_dataset_params
         data_dir = params_config.get("data_dir")
+        if data_dir:
```

Comment:
for job run in AML, we don't need download them. Instead, we can directly mount it?

Reply:
This would be a little bit tricky. Data container has no knowledge of which system is running, local or aml. Even pass itself doesn't know. Any better thought here?

Reply:
Updated